src/org/ods/OdsPipeline.groovy

```diff
@@ -216,18 +216,19 @@ class OdsPipeline implements Serializable {
   }
 
   private void notifyNotGreen() {
-    if (context.notifyNotGreen) {
-      script.emailext(
-        body: '${script.DEFAULT_CONTENT}', mimeType: 'text/html',
-        replyTo: '$script.DEFAULT_REPLYTO', subject: '${script.DEFAULT_SUBJECT}',
-        to: script.emailextrecipients([
-          [$class: 'CulpritsRecipientProvider'],
-          [$class: 'DevelopersRecipientProvider'],
-          [$class: 'RequesterRecipientProvider'],
-          [$class: 'UpstreamComitterRecipientProvider']
-         ])
-      )
-    }
+    String subject =  "Build $context.componentId on project $context.projectId  failed!"
```

Comment:
add

``` if (context.notifyNotGreen) {  ```

Reply:
ok, added in catch block in org.ods.OdsPipeline#execute (as this is the version we use in the opendevstack installation at our site.)